src/include/sof/audio/multiband_drc/multiband_drc.h

```diff
@@ -0,0 +1,84 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
just curious why we need this header? do we plan to include it other places?

Reply:
Because we still need to share some definitions between multiband_drc and multiband_drc_generic? I haven't upload multiband_drc_generic.c yet.

Reply:
ah got it 👍